satzung.md

```diff
@@ -70,7 +70,21 @@ Zu den Aufgaben des Fachschaftsrates gehören insbesondere:
 (5) Näheres regelt die Geschäftsordnung des Fachschaftsrates.
 
 
+## § 4c Fachschaftsrats-Assoziierte
+
+(1) Fachschaftsrats-Assoziierte sind Personen, die den Fachschaftsrat in bestimmten Aufgaben unterstützen. Dies ist nicht möglich bei Ämtern und Aufgaben, die nach der Geschäftsordnung des Fachschaftsrates von dessen Mitgliedern übernommen werden müssen.
+
+(2) Fachschaftsrats-Assoziierte werden durch den Fachschaftsrat benannt.
```

Comment:
```suggestion
(2) Fachschaftsrats-Assoziierte werden durch den Fachschaftsrat benannt. Dazu beschließt der Fachschaftsrat, dass sich die zu benennden Personen und sich die Aufgaben, die diese Personen übernehmen, eignen.
```

Reply:
Hier wird erstens festgelegt, dass es ein Beschluss ist, womit das Ganze veröffentlicht werden muss, und es wird darauf eingegangen, dass der FSR auch die Aufgabe für geeignet hält. Das soll das mit EvaP ausschließen.

Reply:
Aber impliziert die Benennung von FSR-Assoziierten nicht ohnehin schon, dass der FSR diese Aufgaben für abgebbar hält?

Reply:
Löst sich das nicht durch den vorhandenen Wortlaut "in _bestimmten_ Aufgaben"?